### PR TITLE
tbb: add static libs

### PR DESCRIPTION
**Pull request by jeroen**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

We need these for R to build the opencv bindings. Thanks!

### CODE REVIEW

File: Formula/tbb.rb [hunk 1]
```diff
@@ -32,6 +33,10 @@ def install
       system "cmake", "..", *args
       system "make"
       system "make", "install"
+      system "make", "clean"
+      system "cmake", "..", *args, "-DBUILD_SHARED_LIBS=OFF"
+      system "make"
+      lib.install Dir["**/*.a"]
```
Comment (carlocab) [inline Formula/tbb.rb:39]: How much more complicated would it be to be more specific here?

File: Formula/tbb.rb [hunk 2]
```diff
@@ -32,6 +33,10 @@ def install
       system "cmake", "..", *args
```
Comment (carlocab) [inline Formula/tbb.rb:33]:
```suggestion
      system "cmake", "..", *args, "-DCMAKE_INSTALL_RPATH=#{rpath}"
```
This should fix the linkage warnings, I think.

File: Formula/tbb.rb (hunk 1 above)
Comment (jeroen) [reply Formula/tbb.rb:39]: The full path looks like this:

```
./oneTBB-2021.5.0/build/appleclang_12.0_cxx11_64_release/libtbb.a
./oneTBB-2021.5.0/build/appleclang_12.0_cxx11_64_release/libtbbmalloc.a
```

File: Formula/tbb.rb (hunk 2 above)
Comment (jeroen) [reply Formula/tbb.rb]: ok committed.

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**